pages/osx/say.md

```diff
@@ -1,19 +1,23 @@
 # say
 
-> Uses text-to-speech to speak through the default sound device.
+> Converts text to speech.
 
 - Speak a phrase aloud:
 
-`say "I like to ride my bike."`
+`say {{"I like to ride my bike."}}`
 
 - Speak a file aloud:
 
-`say -f {{filename}}`
+`say -f {{filename.txt}}`
 
-- Create an AAC compressed audio file with the spoken text:
+- Speak a phrase with a custom voice and speech rate:
```

Comment:
I was about to change everything to "say", but it sounds really awkward on some of them ("Say a file aloud" is awkward and "Say the contents of a file aloud" is wordy and still a bit awkward).

To my ear, it tends to sound wrong if the object of the verb "say" is anything but a literal quotation; i.e. "say 'hello'" is fine, but "say a greeting" sounds really weird, "speak a phrase" sounds okay, but "say a phrase" is a bit uneasy. There are exceptions ("say nice things about me" and "say what you need to say" both sound less weird), but for the most part, I think a verb like "speak" or "utter" sounds more natural.

Another approach would be "Read a message aloud", "Read a message aloud with a custom voice and speech rate", etc. That wording sounds very natural and comfortable to me (though maybe others might not realize I mean "read" like "read a book" and not "read a file into memory"). What do you guys think?